src/importers/jiraCsv/JiraCsvImporter.ts

```diff
@@ -62,15 +183,50 @@ export class JiraCsvImporter implements Importer {
     }
 
     for (const row of data) {
-      const url = this.organizationName
-        ? `https://${this.organizationName}.atlassian.net/browse/${row['Issue key']}`
-        : undefined;
-      const mdDesc = j2m.to_markdown(row['Description']);
+      let mdDesc: string = j2m.to_markdown(row['Description']);
+
+      // add attachment links to description
+      let jiraAttachments = row.Attachment || [];
+      if (jiraAttachments && typeof jiraAttachments === 'string') {
+        jiraAttachments = [jiraAttachments];
+      }
+      if (jiraAttachments.length) {
+        mdDesc += `${mdDesc && '\n\n'}Attachment(s):`;
+      }
+      jiraAttachments.forEach((att: string) => {
+        if (att) {
+          const [, userId, linkName, linkUrl] = att.split(';');
+          mdDesc += `\n[${linkName}](${linkUrl})`;
+          if (!importData.users[userId]) {
+            importData.users[userId] = {
+              name: userId,
+            };
+          }
+        }
+      });
+
+      // put epic link
+      mdDesc +=
+        row['Custom field (Epic Link)'] && baseUrl
+          ? `${mdDesc && '\n\n'}[View epic link in Jira \[${
+              row['Custom field (Epic Link)']
+            }\]](${baseUrl}/${row['Custom field (Epic Link)']})`
+          : '';
```

Comment:
This could be written in easier to read form:

```
if (row['Custom field (Epic Link)'] && baseUrl) {
  mdDesc += `${mdDesc}\n\n[View epic link in Jira \[${
              row['Custom field (Epic Link)']
            }\]](${baseUrl}/${row['Custom field (Epic Link)']})`
}